apps/remix-ide/src/app/tabs/test-tab.js

```diff
@@ -242,13 +264,30 @@ module.exports = class TestTab extends ViewPlugin {
     } else if (result.type === 'testFailure') {
       if (result.hhLogs && result.hhLogs.length) this.printHHLogs(result.hhLogs, result.value)
       if (!result.assertMethod) {
+        let debugBtn = yo``
+        if (result.errMsg.includes('Transaction has been reverted by the EVM')) {
```

Comment:
I think the option to debug should be displayed in all cases (e.g if tests fails or not)

Reply:
after discussion, this will be handled in an another PR.

Reply:
Issue to track: https://github.com/ethereum/remix-project/issues/1562